kernel/sched.c

```diff
@@ -1599,3 +1599,13 @@ int z_sched_waitq_walk(_wait_q_t  *wait_q,
 
 	return status;
 }
+
+/* This routine exists for benchmarking purposes. It is not used in
```

Comment:
Would it be worth it to guard this with a Kconfig?

Reply:
In a previous version, I had that guarded by a #ifdef CONFIG_TEST/#endif.  Andy correctly pointed out that the build system would automatically remove it from the build so there was little point in the extra guard. Since I still wanted to ensure that this routine is not one that should be making it into general use, I figured a good compromise would be to replace it with a comment as someone, somewhen is going to question why the routine is there.